config/metrics/metrics-service.yaml

```diff
@@ -3,6 +3,8 @@ kind: Service
 metadata:
   name: etcd-shield-metrics
   namespace: etcd-shield
+  labels:
+    apps: etcd-shield
```

Comment:
```suggestion
    app: etcd-shield
```

Reply:
the service monitor has `apps: etcd-shield`. other resources have `app: etcd-shield`. I'll set the latter one for everything

Reply:
done at d95b03b96b154e5ab8db30ed4ff38951d012e805